### ISSUE TITLE
No margin between sibling table elements

### ISSUE THREAD

**Issue by barrymcgee** [canonical-web-and-design/vanilla-framework#2491]
**Describe the bug**

When using two `<table>` elements consecutively, the margins between them collapse.

**To Reproduce**

See Codepen: https://codepen.io/barrymcgee/pen/mdbPKme

**Expected behavior**

There would be a margin of at least 1rem between consecutive tables.

**Comment by lyubomir-popov** [comment]
Isn't this a duplicate of [this](https://github.com/canonical-web-and-design/vanilla-framework/issues/2420)?

The reason it isn't fixed yet is percy shows a difficult to reproduce bug on old versions of firefox where the trs get extra white space, which increases on resize.

A workaround using transparent borders was [rejected](https://github.com/canonical-web-and-design/vanilla-framework/pull/2422).

So I would say this is blocked on making a decision about the ff bug.

**Comment by lyubomir-popov** [comment]
From our latest conversation with @deadlight, I think we discussed changing padding to margin then filing a separate issue to deal with the firefox issue if it surfaces. THat would mean ignoring the percy diff though, not sure if everyone is ok with that.
